### PR TITLE
ImGui C++ bindings for ANSI Escape Sequence-based text coloring

### PR DESCRIPTION
**Pull request by connorjak**
`text_ansi()` and `text_ansi_colored()` are added. They have the exact same interface as `text()` and `text_colored()`, with the exception of logic going on inside to find and react to ANSI escape sequences. I find this useful for displaying logged colored terminal output. `colorama.Fore` colors should all work; no `init()` required.

I do not know if adding ANSI escape sequences and using `text_ansi()` is computationally slower than a series of `text_colored()` calls. That would be interesting to figure out for use cases besides having ANSI codes already embedded in the input.

C++ implementation from here: https://gist.github.com/ddovod/be210315f285becc6b0e455b775286e1

Sorry for no tests yet; I haven't been able to get the test build configuration correct yet.

### CODE REVIEW

**Comment by swistakm** [comment]
Thanks. Could you take a look at builds failing on AppVeyor?

File: AnsiFeed/AnsiTextColored.cpp [hunk 1]
```diff
@@ -0,0 +1,562 @@
...
+        }
+
+        const char* seqEnd = &s[2];
+        while (*seqEnd != 'm') {
+            seqEnd++;
+        }
+
+        std::string seq( &s[2], seqEnd );
+        std::string colorStr;
+        for (const auto& el : jet::split(seq, ";")) {
```
Comment (connorjak) [inline AnsiFeed/AnsiTextColored.cpp:68]: CI build is failing here on the `const auto&`. This syntax is probably too new; it should be replaced with a less-modern substitute.

**Comment by connorjak** [comment]
Otherwise, looks like there's some sort of deprecation thing going on with Sphinx? I can't make sense of the other errors.

**Comment by connorjak** [comment]
These changes might have fixed the tests-build on my computer. I'll get on to writing some tests soon.

**Comment by swistakm** [comment]
Sorry it took so long, when the pipelines finish I will release it as new version. I made slight changes to the file names to keep everything more consistent with existing code.

**Comment by swistakm** [comment]
I had to spend 6 hours on investigating CI issues. Seems like we're almost there. But in the meantime I've noticed that this relies on some ImGui internals that have some risk of being changed often. That's I will make a small change to expose it in extra submodule. This way users will be aware that this isn't something available out-of-the box in the base library.

Does it sound reasonable?

**Comment by connorjak** [comment]
Sure, that sounds reasonable. I think the implementation I used (linked above) copied the internals of imgui's text() function, so while it should be rather straightforward to keep up to date, and the API should stay rock-solid, it needs to be maintained.

**Comment by swistakm** [comment]
Released as `1.3.0`:
- [release notes](https://github.com/swistakm/pyimgui/releases/tag/1.3.0)
- [available distributions](https://pypi.org/project/imgui/1.3.0/#files)

**Pull request merged**